src/main/java/de/guntram/mcmod/durabilityviewer/client/gui/GuiItemDurability.java

```diff
@@ -183,6 +184,9 @@ public void onRenderGameOverlayPost(DrawContext context, float partialTicks) {
         ItemIndicator[] trinkets;
         if (haveTrinketsApi) {
             List<ItemStack> equipped = getTrinkets(player);
+            if (equipped == null) {
+                equipped = Collections.emptyList();
```

Comment:
```suggestion
                equipped = List.of();
```

this does basically the same, but you don't need a new import for that (and could add new elements to it, if you ever need them).